tutorials/data-science/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Data Science on Google Cloud Platform
+description: Use Google Cloud Platform tools to automate, share, and scale Data Science workflows.
+author: jerjou
+tags: Data Science
+date_published: 2017-05-10
+---
+Data Science - a process for gleaning insight from data, and using said data to
+perform actions - comprises a number of steps, any (or all) of which can aided
+by tools provided by the Google Cloud Platform.
```

Comment:
I suggest breaking this into two sentences, because the parenthetical clause is long enough that it hinders the flow of the main point.